release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,16 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 29 February 2024 - New Feature - Admin - Organization keys added [ID_38944]
```

Comment:
This RN has already been added by #2802, so it should probably be removed from this PR.

Reply:
Removed